src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -135,22 +135,14 @@ public static void main(String[] args) {
 			public void run() {
 				Application app = new Application("Jobs 2D");
 
-				DrawerFeature drawerFeature = new DrawerFeature();
-				drawerFeature.setup(app,app.getFreePanel(),DriverFeature.getDriverManager());
-
-				CommandsFeature commandsFeature = new CommandsFeature();
-				commandsFeature.setup(app,app.getFreePanel(),DriverFeature.getDriverManager());
+				new DrawerFeature(app);
+				new CommandsFeature();
+				new DeviceUsageFeature();
+				new RecordingFeature(app,DriverFeature.getDriverManager());
```

Comment:
Now this code looks very strange. Now each feature instance is "lost" after construction. It can cause some problems with other functions.
I suggested: " Try adding features manager facility for adding the features and configuring them all at once. ". It would involve:
1. Creating a class that manages features ("manager").
2. New instances of features would be added to the "manager".
3. Calling some method (e.g., configure or setup) on the "manager" would execute setup on the added features.
4. The "manager" could be e.g., global static or singleton or available from Application instance
5. Accessing the added features should be possible from the "manager"
6. It would be safest to devoid particular features from static (global) methods to make sure all other functions access features or their components correctly.